1.9/overview/networking/service-discovery/index.md

```diff
@@ -0,0 +1,20 @@
+---
+post_title: Service Discovery in DC/OS
+nav_title: Service Discovery
+menu_order: 2
+---
+
+DC/OS comes with a highly available, distributed, DNS-based service discovery. There are two main components to the service discovery mechanism in DC/OS. A centralized component called [mesos-dns][4], which runs on every master, and a distributed component called [spartan][6] that runs on every agent. 
```

Comment:
Global: spartan should be called the `DNS Forwarder (Spartan)` per the components page. Or at least on its initial occurrence.